src/store/getters/state-getters.ts

```diff
@@ -468,7 +468,8 @@ export const isDissolutionValid = (state: StateIF): boolean => {
   const isCourtOrderValid = isRoleStaff(state) ? getCourtOrderStep(state).valid : true
   const isStaffPaymentValid = isRoleStaff(state) ? getStaffPaymentStep(state).valid : true
 
-  const isEffectiveDateTimeValid = (isBaseCompany(state) || isTypeFirm(state)) ? getEffectiveDateTime(state).valid : true
+  const isEffectiveDateTimeValid = (isBaseCompany(state) ||
+  isTypeFirm(state)) ? getEffectiveDateTime(state).valid : true
```

Comment:
Unfortunately lint doesn't catch this, but please indent continuing lines.

By the way, an easy-to-read format for ternaries is like this:
```
const isEffectiveDateTimeValid = (isBaseCompany(state) || isTypeFirm(state))
  ? getEffectiveDateTime(state).valid
  : true
```